CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ and includes an additional section for migration notes.
 - *ORCA-784* Changed documentation to replace restore with copy based on task's naming as well as changed file name from `website/docs/operator/restore-to-orca.mdx` to `website/docs/operator/reingest-to-orca.mdx`.
 - *ORCA-724* Updated ORCA recovery documentation to include recovery workflow process and relevant inputs and outputs in `website/docs/operator/data-recovery.md`.
 - *ORCA-789* Updated `extract_filepaths_for_granule` to more flexibly match file-regex values to keys.
+- *ORCA-787* Updated `modules/api-gateway/main.tf` to remove extra `orca` in path to avoid confusion.
```

Comment:
Maybe update to "Modified api gateway stage name to remove the extra `orca` from the data management URL path"